### ISSUE TITLE
subscript and superscript cytoscape

### ISSUE THREAD

**Issue by eagomezc** [cytoscape/RCy3#64]
I've been using RCy3 lately and I've found it a very useful tool. Thanks for keeping it updated. 

I haven't found thou if it's possible, through RCy3, creates labels with subscript or superscript. I've been working with some lipidomics data and it's very important for me to keep the names of each lipid as it is shown in the literature. 

Is there a way to apply subscript to the labels in cytoscape using RCy3?

Thanks for the answer you can give me.

**Comment by AlexanderPico** [comment]
If it is something you can do in Cytoscape directly, then we can probably do it in RCy3. Do you know if this is supported by Cytoscape? Can you provide an example?

**Comment by AlexanderPico** [comment]
Well, I learned something new today!  You *can* enter subscripted numbers into labels in Cytoscape (and RCy3).  In order to do it on the mac, I had to add Unicode to my keyboard input sources. Then pressing Option while entering Unicode values will generate special characters like subscripts. The subscripted 2, e.g., is <option>2082. Using this I could type subscripted 2 characters directly in to Cytoscape's node table and into R scripts. Using `setNodeLabelBypass()` I could demonstrate that RCy3 correctly passes these Unicode characters through cyREST to Cytoscape, like so:

```
openSession() #to open sample session file
setNodeLabelBypass('YNL216W','RAP₂')  #to change label of "RAP1" to "RAP₂"
```
